ui/src/builder/BuilderComponentShortcuts.vue

```diff
@@ -109,6 +109,7 @@
 			</div>
 			<div
 				class="actionButton delete"
+				data-action="delete"
```

Comment:
Any chance we can use some sort of namespace like `data-automation-action` or is that not configurable?

Reply:
Good idea! We can name it however we want so I will change this.